### PR TITLE
Mass mail

### PR DESCRIPTION
**Pull request by johnEthicalTechnology**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -117,7 +117,7 @@ select {
 }
 
 button {
-  max-width: 29%;
+  max-width: 100%;
```
Comment (jeznag) [inline src/index.css:120]: This may affect the buttons for the mailing list download

File: src/utils/emailAndIdExtract.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  email: string
+  id: string | number
+}
+
+export default function emailAndIdExtract (results: UnprocessedResultsFromCRM[]) {
+    const propertyObjects = results.slice(1)
+
+    const emailsAndIds = propertyObjects.reduce((resultsArray: MassMailObject[], property: UnprocessedResultsFromCRM) => {
+        if (typeof property.owner_details[0].Email === 'string') {
+            resultsArray.push(Object.assign({},
```
Comment (jeznag) [inline src/utils/emailAndIdExtract.ts]: is object.assign needed here?

File: zoho-objects/zoho-results-small.ts [hunk 3]
```diff
@@ -0,0 +1,1654 @@
+export const results: any = [{
```
Comment (jeznag) [inline zoho-objects/zoho-results-small.ts]: Is this file needed?

File: src/index.css (hunk 1 above)
Comment (johnEthicalTechnology) [reply src/index.css:120]: They seem to be maintaining their regular width.

File: src/utils/emailAndIdExtract.ts (hunk 2 above)
Comment (johnEthicalTechnology) [reply src/utils/emailAndIdExtract.ts]: I thought it would be b/c I need to create a new object every time an email is found.

File: zoho-objects/zoho-results-small.ts (hunk 3 above)
Comment (johnEthicalTechnology) [reply zoho-objects/zoho-results-small.ts]: It was just for testing. I can get rid of it.

File: src/utils/emailAndIdExtract.ts (hunk 2 above)
Comment (jeznag) [reply src/utils/emailAndIdExtract.ts]:
``` 
{
  email: property.owner_details[0].Email,
  id: property.owner_details[0].id
}
``` 

creates a new object already. You'd use Object.assign if there was an existing object that you wanted to clone.

Comment (johnEthicalTechnology) [reply src/utils/emailAndIdExtract.ts]: Okay, that makes sense!
